### DISCUSSION TITLE
Google Sites Now Works

### DISCUSSION THREAD

**Discussion by allancoding** [EmulatorJS/EmulatorJS#610] [General]
Here is an example I have made in google sites: [EmulatorJS Google Sites Example](https://sites.google.com/view/emulatorjs/home)
- It is recommended if you game file is over a mb then to host your ROM on a different server and not base64.
- You can add a full page embed with custom code
- You can use [DriveToWeb](https://www.drv.tw/) if you want to host your files using google drive (as show in my working example)
Here is the code used in the example game:
```html
<html>
    <head>
        <style>
        body, html {
            margin: 0;
            padding: 0;
        }
        </style>
    </head>
    <body>
        <div style="width:100%;height:100%;max-width:100%">
            <div id="game"></div>
        </div>
        <script>
            EJS_player = "#game";
            EJS_core = "snes";
            EJS_gameName = "Secret of Mana";
            EJS_color = "#0097c4";
            EJS_pathtodata = "https://cdn.emulatorjs.org/latest/data/";
            EJS_gameUrl = "https://allancoding-website-files.on.drv.tw/files/ROMS/Secret of Mana.zip";
        </script>
        <script src="https://cdn.emulatorjs.org/latest/data/loader.js"></script>
    </body>
</html>
```

**Comment by calicoplayz99** [comment]
what site do you recommend if you cant use DriveToWeb?

**Reply by ethanaobrien** [reply]
Vultr/digitalocean/heroku

Do keep in mind that (in the first place) I do not support drive to web

also keep in mind publicly sharing copyrighted content is not legal can get you in trouble

**Comment by aunknownschoolkid** [comment]
Hey,

when I try and run the game, I get a "Network Error" how do you fix this

**Reply by SimonTheCat128** [reply]
Me too

**Reply by qwertyzx343** [reply]
me three
